### PR TITLE
IAM deprecation update

### PR DESCRIPTION
**Pull request by chewbeccy**

### CODE REVIEW

File: docs/meshstack.identity-federation.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ graph LR;
 ## Designing Multi-Cloud Identity and Access Management
 
 Designing an effective multi-cloud identity and access management strategy is no small feat. This guide describes the
-most important considerations that enterprise architects and IAM specialists need to be aware of when designing
+most important considerations that nterprise architects and IAM zpecialists need to be aware of when designing
```
Comment (sttomm) [inline docs/meshstack.identity-federation.md:53]: f: Hmm, those 2 words are broken now

**Review by sttomm — APPROVED** [review]
lgtm except for 2 broken words

File: docs/meshstack.identity-federation.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ id: meshstack.identity-federation
 title: Identity and Access Management
 ---
 
-One of meshStacks main features is to provide single identities across multiple cloud and container platforms. Therefore, meshStack contains a full-featured **Identity Broker** (meshIdB) which consumes identities from company directories and adds authorization information for the specific cloud platforms.
+meshStack supports a central Identity Provider architecture across multiple cloud and container platforms. Therefore, meshStack contains a full-featured Identity Broker (meshIdB) which consumes identities from company directories and adds authorization information for the specific cloud platforms.
```
Comment (JohannesRudolph) [inline docs/meshstack.identity-federation.md:6]: meshIdB adding authorization info is only relevant to the deprecated meshIdB based K8s, OpenStack, AWS integrations. 

I think it makes most sense to redesign this whole page and remove `meshStack-provisioned Identities` as a concept from the documentation.

File: docs/meshstack.identity-federation.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ The identity provisioning mechanism and the resulting federation setup depends o
 
 ### meshStack-provisioned Identities
 
-In this mode meshStack provisions identities used in meshProjects on the respective cloud platform. The lifecycle of the identity is automatically managed via meshStack, including [revocation and deprovisioning](./meshstack.user-revocation.md). In this mode, identity federation involves the meshIdB and meshcloud can provide all necessary configuration and setup assistance.
+In this mode meshStack provisions identities used in meshProjects on the respective cloud platform. The lifecycle of the identity is automatically managed via meshStack, including [revocation and deprovisioning](./meshstack.user-revocation.md). In this mode, identity federation involves the meshIdB and meshcloud can provide all necessary configuration and setup assistance. This is only available for Cloud Foundry.
```
Comment (JohannesRudolph) [inline docs/meshstack.identity-federation.md:64]: This is already explained in the table below, not sure it makes sense to duplicate

File: docs/meshstack.identity-federation.md [hunk 4]
```diff
@@ -126,7 +126,7 @@ meshStack supports the following identity provisioning strategies:
 | :-----------: | :-------------------------------------------------------------------: | :------------------------------------------------------------------------: |
 |   OpenStack   |         [deprecated](./meshstack.openstack.index.md#meshidp)          |         [supported](./meshstack.openstack.index.md#enterprise-idp)         |
 | Cloud Foundry |    [supported](meshstack.cloudfoundry.index.md#uaa-configuration)     |                                 _planned_                                  |
-|  Kubernetes   | [supported](./meshstack.kubernetes.index.md#api-server-configuration) |                                 _planned_                                  |
+|  Kubernetes   | [deprecated](./meshstack.kubernetes.index.md#api-server-configuration) |                                 _planned_                                  |
```
Comment (JohannesRudolph) [inline docs/meshstack.identity-federation.md:129]: note: externally provisioned is supported (this is missing!)

I think I made that change in https://github.com/meshcloud/meshcloud-docs/pull/646/files

**Review by JohannesRudolph — APPROVED** [review]
some general remarks, nothing that blocks already merging these improvements

File: docs/meshstack.identity-federation.md (hunk 2 above)
Comment (chewbeccy) [reply docs/meshstack.identity-federation.md:6]: I agree that the page needs a general redesign. For the sake of getting important improvements on the page we will do the overhaul another time

**Pull request merged**